compiler/optimizer/LoopReducer.cpp

```diff
@@ -980,8 +980,8 @@ TR_LoopReducer::generateArraycopy(TR_InductionVariable * indVar, TR::Block * loo
    if (storeNode->getType().isBCD() &&
        !(storeSize == 1 || storeSize == 2 || storeSize == 4 || storeSize == 8))
       {
-      dumpOptDetails(comp(), "arraycopy storeNode %p is a BCD type (%d) and the storeSize (%d) is not 1,2,4 or 8 so do not reduce arraycopy\n",
-         storeNode,storeNode->getDataType(),storeSize);
+      dumpOptDetails(comp(), "arraycopy storeNode %p is a BCD type (%s) and the storeSize (%s) is not 1,2,4 or 8 so do not reduce arraycopy\n",
```

Comment:
Is the second %s correct?  It looks like that should still be %d as storeSize is an integer.